pages/blog/[...path].vue

```diff
@@ -75,7 +75,7 @@ const formatDate = (date?: string) => {
 
             <div class="mt-8 mx-auto">
                 <time v-if="post.created_at" :datetime="post.created_at" class="text-gray-500">{{
-        formatDate(post.created_at) }}</time>
+                    formatDate(post.created_at) }}</time>
```

Comment:
This should be left to the code linter to handle, not done manually

The code linter does it badly but I'll replace it later

Reply:
sure thing, boss!